packages/core/contracts/merkle-distributor/implementation/MerkleDistributor.sol

```diff
@@ -170,9 +145,8 @@ contract MerkleDistributor is Ownable {
      *         will revert. It also reverts when `_claim`'s `amount` exceeds `remainingAmount` for the window.
      * @param _claim claim object describing amount, accountIndex, account, window index, and merkle proof.
      */
-    function claim(Claim memory _claim) external {
+    function claim(Claim memory _claim) external virtual override {
         _verifyAndMarkClaimed(_claim);
-        merkleWindows[_claim.windowIndex].remainingAmount -= _claim.amount;
```

Comment:
Moved logic to decrement `remainingAmount` to `_verifyAndMarkClaimed` so future devs don't forget to decrement when claiming a leaf